lib/adapter/freshheads/DefaultStackAdapter.ts

```diff
@@ -96,7 +97,8 @@ export default class DefaultStackAdapter implements Adapter {
                         automaticNameDelimiter: '-',
                     },
                 })
-            );
+            )
+            .add(new MinimizationAdapter());
```

Comment:
@gnoesiboe enige nadeel is nu wel dat stel je CSS uit hebt staan hij toch die minimizer toe gaat voegen. Kan geen kwaad maar toch. Je zou hier wel een setting aan mee kunnen geven welke minimizer geladen mag worden maar maakt het in die adapter wel een if statement festijn. Kunnen we zeggen dit is gewoon onze default stack dus deal er maar mee of zou je dat toch willen gaan configureren ?

Reply:
prima inderdaad